### PR TITLE
Created camera tagging module

### PR DESCRIPTION
**Pull request by t-nana**

### CODE REVIEW

File: samples/official/camera-tagging/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# CameraTaggingModule
+
+## Overview
+
+This repository contains the modules necessary to allow individuals to capture images from an RTSP stream, tag them, and upload them to train a Custom Vision model or simply to a Blob store.
+
+It reads 9 environment variables from which 3 are used to build the default URI to the RTSP stream (i.e. rtsp://<RTSP_IP>:<RTSP_PORT>/<RTSP_PATH>), and 4 are used to describe the local blob store. These environment variables can be specified in the docker deployment config.
```
Comment (varunpuranik) [inline samples/official/camera-tagging/README.md]: What do you mean by docker deployment config?

File: samples/official/camera-tagging/deployment.template.json [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+              "REACT_APP_LOCAL_STORAGE_MODULE_NAME": {
+                "value": "azureblobstorageoniotedge"
+              },
+              "REACT_APP_LOCAL_STORAGE_PORT": {
+                "value": "11002"
+              },
+              "REACT_APP_LOCAL_STORAGE_ACCOUNT_NAME": {
+                "value": "camerataggingstorage"
+              },
+              "REACT_APP_LOCAL_STORAGE_ACCOUNT_KEY": {
```
Comment (varunpuranik) [inline samples/official/camera-tagging/deployment.template.json]: Please remove

File: samples/official/camera-tagging/deployment.template.json [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "settings": {
+              "image": "mcr.microsoft.com/azure-blob-storage:latest",
+              "createOptions": {
+                "Env":[
+                  "LOCAL_STORAGE_ACCOUNT_NAME=camerataggingstorage",
+                  "LOCAL_STORAGE_ACCOUNT_KEY=lTkMQ4UZS1cNAuy4P9wthUWxU3+e5OwsMjIp43PbVbNzaHH6I7UVbo+Fx1SuKKeRUf0nvTmh5NqjlS1FyOIt/g=="
```
Comment (varunpuranik) [inline samples/official/camera-tagging/deployment.template.json]: sanitize

File: samples/official/camera-tagging/deployment.template.json [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+          "azureblobstorageoniotedgeToIoTHub": "FROM /messages/modules/azureblobstorageoniotedge/outputs/* INTO $upstream"
+        },
+        "storeAndForwardConfiguration": {
+          "timeToLiveSecs": 7200
+        }
+      }
+    },
+    "VisionSampleModule": {
+      "properties.desired": {
+        "ModelZipUrl": "https://nandhinistorage.blob.core.windows.net/windowwall/Train.VAIDK.zip",
```
Comment (varunpuranik) [inline samples/official/camera-tagging/deployment.template.json]: remove

File: samples/official/camera-tagging/deployment.template.json [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+    "azureblobstorageoniotedge":{
+      "properties.desired": {
+        "deviceAutoDeleteProperties": {
+          "deleteOn": false,
+          "retainWhileUploading": true
+        },
+        "deviceToCloudUploadProperties": {
+          "uploadOn": true,
+          "uploadOrder": "OldestFirst",
+          "cloudStorageConnectionString": "DefaultEndpointsProtocol=https;AccountName=nandhinistorage;AccountKey=jafIBZlc1JE+8LdC/V43UfKsvodIUqYFFCUJHuEK1EpFACB1o9/AS4ym0kjP2QX6x2fEasrBph9UZsbwoHvFew==;EndpointSuffix=core.windows.net",
```
Comment (varunpuranik) [inline samples/official/camera-tagging/deployment.template.json]: Sanitize

File: samples/official/camera-tagging/README.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+## Build and push the Client and Server Container Images
+
+1. Copy over all the folders from **CameraTaggingModule/Arm32Base/** to a folder on an arm32v7 device.
+
+1. Using a command prompt, cd into TaggingClientDocker and run
```
[the remark this answers is not in the document]
Comment (t-nana) [reply samples/official/camera-tagging/README.md]: I believe that's just the convention for sequential numbering in markdown. If you look at the preview of the Readme the numbers increase appropriately.

File: samples/official/camera-tagging/README.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+    - `docker build -t <REGISTRY_NAME>.azurecr.io/<IMAGE_NAME>:<VERSION> .`
+
+1. Push the image to your repository
+    - Make sure to first login using `docker login -u <REGISTRY_USER_NAME> -p <REGISTRY_PASSWORD> <REGISTRY_NAME>.azurecr.io`
+    - Then push the image using the command `docker push <REGISTRY_NAME>.azurecr.io/<IMAGE_NAME>:<VERSION>`
+
+1. Repeat steps 2 & 3 for TaggingServerDocker.
```
[the remark this answers is not in the document]
Comment (t-nana) [reply samples/official/camera-tagging/README.md]: Yes for arm32 you still build 2 separate images for the client and server and then combine them at the end.

File: samples/official/camera-tagging/deployment.template.json (hunk 2 above)
Comment (t-nana) [reply samples/official/camera-tagging/deployment.template.json]: Removed

File: samples/official/camera-tagging/README.md (hunk 1 above)
Comment (t-nana) [reply samples/official/camera-tagging/README.md]: Changed to deployment.template.json.

**Pull request closed without merging**